docs/_docs/02_features/engine.md

```diff
@@ -67,9 +73,9 @@ engine {
 ### Parameters
 
 * `source`: (Required) The source of the plugin. Multiple engine approaches are supported, including GitHub repositories, HTTP(S) paths, and local absolute paths.
-* `version`: (Required for GitHub) The version of the plugin to download from GitHub releases.
+* `version`: The version of the engine to download from GitHub releases, if not specified, the latest release is downloaded.
```

Comment:
```suggestion
* `version`: The version of the engine to download from GitHub releases, if not specified, the latest release is always downloaded.
```

We should be clear about what happens if users already have a latest version installed.